torch/_export/serde/schema.py

```diff
@@ -10,10 +10,28 @@ class _Union:
     @classmethod
     def create(cls, **kwargs):
         assert len(kwargs) == 1
-        return cls(**{**{field.name: None for field in fields(cls)}, **kwargs})
+        obj = cls(**{**{f.name: None for f in fields(cls)}, **kwargs})
+        return obj
+
+    @classmethod
+    def fields(cls):
```

Comment:
This name is a bit confusing. It returns a dataclass?

Reply:
Yeah...I don't know of a good way to do this, but I want something like referencing `Argument.fields().as_tensor` to give us `"as_tensor"` so that I can compare it against some `Argument.type` value which is the field name (`"as_tensor"`). This is for the case statement in `import_input()`. An alternative is to compare `Argument.type` to the string `"as_tensor"` itself, but I wasn't sure if that was a good idea...

Reply:
wait sorry I forgot about enums...🤦‍♂️